### ISSUE TITLE
Chisel in a mechanical user right click causes crash loop

### ISSUE THREAD

**Issue by itanshi** [rwtema/extrautilities_old_issues#1155]
Was probably a bad idea :/

chisel 0.0.7.7
extra utilities2 1.1.1

Caused by: java.lang.RuntimeException: Error while interacting with block minecraft:air with 1xitem.chisel.chisel_diamond@0. Method:GENERIC_CLICK Button:LEFT_CLICK

**Comment by ghost** [comment]
No, it's an automatic chisel multiblock. It's a clever idea and the crash is a bug.
I'd suggest getting a world backup from before the crash and stopping it from happening. Alternatively, updating EU2 might disrupt the cycle for now. If not, you can try deactivating the user in the config file long enough to log in and stop the madness. I haven't seen Tema's code, but you could conceivably destroy all the mechanical users in the world doing that.

**Comment by itanshi** [comment]
we restored the region from backup, its ok, ty
